src/commands/sentinel-config.json

```diff
@@ -87,6 +87,7 @@
                         "name":"set",
                         "token":"SET",
                         "type":"block",
+                        "multiple": true,
```

Comment:
please add these changes to the history section

Reply:
the reply schema doesn't need an update, right?

Reply:
Hello @oranagra , I have made the changes and run it locally, the [reply-schemas-validator] passed now. There was issue in already existing code , the type of announce-port was type :integer, I have changed it to string in json file in reply-schema.